packages/fxa-settings/src/pages/ResetPassword/ConfirmResetPassword/en.ftl

```diff
@@ -7,3 +7,5 @@ confirm-pw-reset-header = Reset email sent
 # Instructions to continue the password reset process
 # { $email } is the email entered by the user and where the password reset instructions were sent
 confirm-pw-reset-instructions = Click the link emailed to { $email } within the next hour to create a new password.
+
+resend-pw-reset-banner = Email resent. Add accounts@firefox.com to your contacts to ensure a smooth delivery.
```

Comment:
Instead of including the `accounts@firefox.com` email address as part of the string, could you use a variable instead?

There are multiple reasons this would be preferable: 
* It's possible that translators may mistakenly translate or edit the email address
* There's no way for us to detect if/when the email address is input in correctly
* The string can continue to be used without retranslation even if the email address needs to be updated in the future

Would also want a comment included explaining the variable.
Below example for reference (might want a better variable name?)
```suggestion
# $accountsEmail is the email address the resent password reset confirmation is sent from. (e.g. accounts@firefox.com)
resend-pw-reset-banner = Email resent. Add { $accountsEmail } to your contacts to ensure a smooth delivery.
```